fdbt-aws/sql/h.salesOfferPackage.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE salesOfferPackage(
     `purchaseLocations` varchar(255) DEFAULT NULL,
     `paymentMethods` varchar(255) DEFAULT NULL,
     `ticketFormats` varchar(255) DEFAULT NULL,
+    `isCapped` boolean NOT NULL DEFAULT false,
     INDEX idx_nocCode (nocCode),
```

Comment:
Some requirements missed from the Jira ticket.

Reply:
Also, could we use the ALTER way of doing this, as per other tables. It makes it easier to run the ALTER command on each environment, and also shows that we have changed the table.

Reply:
changes